vendor/modules.txt

```diff
@@ -1,11 +1,11 @@
 # cloud.google.com/go v0.56.0
```

Comment:
everything we update under vendor should be squashed in a single commit, please merge with the following commit.

Reply:
will merge, thanks

Reply:
done